tests/test_integration.py

```diff
@@ -244,4 +258,7 @@ def test_update(
             assert line_number > 80_000
 
             # Check if all states were logged:
-            assert state_list == ["S", "M", "C", "F"]  # , "L"]
+            if launch_name != "interest_launch":
```

Comment:
We don't need this anymore. Other data sets in the future should correctly trigger the landed state (does purple trigger it? you can see in the graph that it does land, but maybe we need to tweak accel values for it too)

Reply:
after your changes, purple launch does not trigger it because the velocity is too wrong, and that can't be fixed because we don't have good rotation data.

For future launches we can edit this file again.